Formula/gcc.rb

```diff
@@ -49,15 +49,6 @@ class Gcc < Formula
   # GCC bootstraps itself, so it is OK to have an incompatible C++ stdlib
   cxxstdlib_check :skip
 
-  if Hardware::CPU.arm?
```

Comment:
Let's put back the patch but make it

    if Hardware::CPU.intel?

Reply:
Done!

Reply:
Thank you!

Reply:
Oops, syntax error. Looking good now, I think.